test/middleware/authentication.middleware.unit.ts

```diff
@@ -24,18 +24,24 @@ const expectedAuthMiddlewareConfig: AuthOptions = {
 
 describe("authentication middleware tests", () => {
 
+  beforeEach(() => {
+    mockAuthMiddleware.mockClear();
+    next.mockClear();
```

Comment:
i would reset the req.originalUrl back to it's original value here so that if we add more tests in the future the url won't still be set to CONFIRMATION_STATEMENT + ACCESSIBILITY_STATEMENT. 
I would set it back to URL ----   req.originalUrl = URL.   The req will be shared between tests as it is declared in the global scope before the tests start.

Reply:
Makes sense. That change should be in now.